### DISCUSSION TITLE
多语言支持 | multilanguage support

### DISCUSSION THREAD

**Discussion by 522247020** [sandbox-bio/sandbox.bio#98] [Q&A] [answered]
在使用网页翻译的情况下，导致终端窗口字符显示混乱，且学习的显示也会出现问题。
希望能提供页面翻译，和学习内容的翻译。
---
In the case of using web page translation, the display of terminal window characters is confused, and the display of learning will also have problems.  
Hope to provide page translation, and learning content translation.

**Comment by robertaboukhalil** [comment, marked as answer]
Hello @522247020, multi-language support is not on the roadmap, and for custom terminal characters, you might need to reach out to the v86 project directly: https://github.com/copy/v86
